### PR TITLE
vlc-telnet component documentation added

### PR DESCRIPTION
**Pull request by rodripf**
**Description:**
Documentation added for the new component vlc-telnet. 

PR in home-assistant: https://github.com/home-assistant/home-assistant/pull/24290

## Checklist:

- [X] Branch: `next` is for changes and new documentation that will go public with the next [home-assistant](https://github.com/home-assistant/home-assistant) release. Fixes, changes and adjustments for the current release should be created against `current`.
- [X] The documentation follows the [standards][standards].

[standards]: https://developers.home-assistant.io/docs/documentation_standards.html

### CODE REVIEW

File: source/_components/vlc-telnet.markdown [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+description: "Instructions on how to integrate VLC media player into Home Assistant using the telnet interface."
+date: 2019-06-02 23:58
+sidebar: true
+comments: false
+sharing: true
+footer: true
+logo: videolan.png
+ha_category:
+  - Media Player
+ha_release: 0.35
```
Comment (klaasnicolaas) [inline source/_components/vlc-telnet.markdown]:
```suggestion
ha_release: 0.95
```

File: source/_components/vlc-telnet.markdown [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+ha_iot_class: Local Polling
+redirect_from:
```
Comment (klaasnicolaas) [inline source/_components/vlc-telnet.markdown]: Please remove the `redirect_from`

File: source/_components/vlc-telnet.markdown [hunk 3]
```diff
@@ -0,0 +1,70 @@
...
+
+The `vlc-telnet` platform allows you to control a [VLC media player](http://www.videolan.org/vlc/index.html) using the built in telnet interface.
+
+To add a VLC media player to your installation, add the following to your `configuration.yaml` file:
+
+```yaml
+# Example configuration.yaml entry
+media_player:
+  - platform: vlc-telnet
+    host: "127.0.0.1"
```
Comment (klaasnicolaas) [inline source/_components/vlc-telnet.markdown]:
```suggestion
    host: IP_ADDRESS
```

File: source/_components/vlc-telnet.markdown [hunk 4]
```diff
@@ -0,0 +1,70 @@
...
+    password: "test"
```
Comment (klaasnicolaas) [inline source/_components/vlc-telnet.markdown]:
```suggestion
    password: PASSWORD
```

File: source/_components/vlc-telnet.markdown [hunk 5]
```diff
@@ -0,0 +1,70 @@
...
+```
+
+{% configuration %}
+name:
```
Comment (klaasnicolaas) [inline source/_components/vlc-telnet.markdown:30]:
```suggestion
name:
  default: VLC-TELNET
```

There is a default config variable

File: source/_components/vlc-telnet.markdown [hunk 6]
```diff
@@ -0,0 +1,70 @@
...
+pasword:
+  description: The password to control the VLC through the telnet interface.
+  required: true
+  type: string
+host:
+  description: The hostname or IP address where the VLC Player is running.
+  required: true
+  type: string
+port:
+  description: The port number where the VLC Player is running. The default value is *4212*
```
Comment (klaasnicolaas) [inline source/_components/vlc-telnet.markdown]: same

**Review by klaasnicolaas — CHANGES_REQUESTED** [review]

File: source/_components/vlc-telnet.markdown [hunk 7]
```diff
@@ -0,0 +1,70 @@
...
+
+This service will control any instance of VLC player on the network with the telnet interface activated. To activate the telnet interface on your VLC Player please read the [official VLC documentation](https://wiki.videolan.org/Documentation:Modules/telnet/).
+
+## {% linkable_title Full configuration %}
+
+A full configuration for VLC could look like the one below:
+
+```yaml
+# Example configuration.yaml entry
+media_player:
```
Comment (klaasnicolaas) [inline source/_components/vlc-telnet.markdown:61]: Here you could do the config without the placeholders 😉

**Review by balloob — APPROVED** [review]

**Pull request merged**